<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_ManagePartitions</class>
 <widget class="QWidget" name="Page_ManagePartitions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="PartitionView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Device</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../minst_resources.qrc">
        <normaloff>:/images/resources/lock_small.png</normaloff>:/images/resources/lock_small.png</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mount Point</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Format</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="margin">
      <number>9</number>
     </property>
     <item>
      <widget class="QPushButton" name="B_GParted_2">
       <property name="text">
        <string>New parition table</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCreatePartition">
       <property name="text">
        <string>New partition</string>
       </property>
       <property name="icon">
        <iconset resource="../minst_resources.qrc">
         <normaloff>:/images/resources/add.png</normaloff>:/images/resources/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button">
       <property name="text">
        <string>Edit partition</string>
       </property>
       <property name="icon">
        <iconset resource="../minst_resources.qrc">
         <normaloff>:/images/resources/edit.png</normaloff>:/images/resources/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDeletePartition">
       <property name="text">
        <string>Delete partition</string>
       </property>
       <property name="icon">
        <iconset resource="../minst_resources.qrc">
         <normaloff>:/images/resources/cancel.png</normaloff>:/images/resources/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonR">
       <property name="text">
        <string>Refresh list</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUnmount">
       <property name="text">
        <string>Unmount</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../minst_resources.qrc"/>
 </resources>
 <connections/>
</ui>
